### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Will-Cooper**
Splitting the functionality into multiple files as well as creating some tests for the new `utils` and `plots` scripts.

### CODE REVIEW

File: tests/test_plots.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    return allphoto, allbands
+
+
+@pytest.fixture(scope='module')
+def test_all_parallaxes(db):
+    assert db
+    allplx = all_parallaxes(db_cs)
+    assert len(allplx)
+    assert type(allplx) == pd.DataFrame
+    return allplx
```
Comment (dr-rodriguez) [inline tests/test_plots.py:56]: Aren't these exactly the same as those in test_utils? If so, they shouldn't be here in test_plots

**Comment by dr-rodriguez** [comment]
Took a quick glance and it looks OK to me, should be much better to manage with the code split into multiple modules. The tests probably need some clean up since I notice tests for utils in test_plots, but I'm OK with approving at this stage.

**Review by dr-rodriguez — APPROVED** [review]

**Comment by dr-rodriguez** [comment]
Actually, another recommendation: have the tests/ directory inside the simple_app/ directory (so simple_app/tests/)
I think that is the more standard approach that packages like astropy use.

**Comment by Will-Cooper** [comment]
> Took a quick glance and it looks OK to me, should be much better to manage with the code split into multiple modules. The tests probably need some clean up since I notice tests for utils in test_plots, but I'm OK with approving at this stage.

Yeah I wasn't sure the best way of running the tests of functions in `plots` which call a `utils` function to generate the necessary dataframes etc?

**Comment by dr-rodriguez** [comment]
> > Took a quick glance and it looks OK to me, should be much better to manage with the code split into multiple modules. The tests probably need some clean up since I notice tests for utils in test_plots, but I'm OK with approving at this stage.
> 
> Yeah I wasn't sure the best way of running the tests of functions in `plots` which call a `utils` function to generate the necessary dataframes etc?

Oh, I see why you did that. But isn't that what scope=session is for? I may be wrong, but I thought a pytest.fixture with scope=session meant that any module in the test session would be able to access the results for that (ie, both test_plots and test_utils), just like scope=module means that any function in the given module can access the results for that function. So you could have a single database connection and set up, whose scope is the full session and is accessed by the individual module tests.

**Pull request merged**